src/app/admin/volunteers/shared/newvolunteer-register-form/newvolunteer-register-form.component.scss

```diff
@@ -0,0 +1,160 @@
+$gapBetweenFieldAndHint: "25px";
+$fontFamily: Roboto, sans-serif;
+
+$modalWindowWidth: 555px;
+$modalWindowHeight: 881px;
+
+mat-grid-tile-header {
+  background: white !important;
+  padding: 0 !important;
+  margin: 0 !important;
+  height: 12px !important;
+
+  & a {
+    padding: 0;
+    font-family: Roboto;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 12px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: rgb(177, 168, 168);
+  }
+}
+
+mat-grid-tile,
+mat-form-field {
+  width: 231px;
+  height: 65px;
+  margin: 0 !important;
+  margin-top: 21px;
+  padding: 0 !important;
+
+  & input {
+    border: 0.5px solid darkblue;
+    background-color: white;
+    margin-top: 10px;
+    padding: 0;
+    height: 40px;
+    width: 230px;
+    text-align: left;
+  }
+}
+
+::ng-deep mat-form-field .mat-form-field-underline {
+  display: none;
+}
+
+::ng-deep mat-form-field .mat-form-field-infix {
+  border-top: none;
+}
+
+mat-select,
+.mat-select {
+  border: 0.5px solid darkblue;
+  background-color: white;
+  margin-top: 10px;
+  padding: 0;
+  height: 40px;
+  width: 229px;
+  vertical-align: middle;
+  line-height: 40px;
+  text-align: left;
+}
+
+mat-grid-list {
+  background-color: rgb(255, 255, 255);
+}
+
+mat-icon {
```

Comment:
this affects all other icons in dialog, for example if you want to add a dialog close icon

Reply:
Yes - its according our vision. So, since <mat-icon> style is not defined globally according to the Figma properties,  we define such one for an exact component item (newvolunteer-register-form) inside scss, to avoid usage of inline styles